src/App.js

```diff
@@ -3,15 +3,18 @@ import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
-  const [newTodo, setNewTodo] = useState(""); // state for newTodo
+  const [todoList, setTodoList] = useState([]);
+
+  function addTodo(newTodo) {
+    setTodoList([...todoList, newTodo]);
```

Comment:
No need to change this at all, but just as a reference to the example in the assignment, this will do the exact same thing. Oh and `prevTodos` is can be anything you like, it's just common practice to call it either `prevState` or `prevWhateverStateYoureReferencing`



```suggestion
    setTodoList((prevTodos) => [...prevTodos, newTodo]);
```